src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: auto;
```

Comment:
redundant width property, you can remove it

Reply:
if I type ( .stars__star:last-child {
   margin-right: 0;
}) then the tests do not pass.

Reply:
Sorry, all corrections)